test/tools/d_do_test.d

```diff
@@ -452,11 +452,6 @@ bool gatherTestParameters(ref TestArgs testArgs, string input_dir, string input_
     if (testArgs.mode == TestMode.FAIL_COMPILE)
         testArgs.requiredArgs = "-verrors=0 " ~ testArgs.requiredArgs;
 
-    // https://issues.dlang.org/show_bug.cgi?id=10664: exceptions don't work reliably with COMDAT folding
-    // it also slows down some tests drastically, e.g. runnable/test17338.d
```

Comment:
What about the show down mentioned by this comment. Is it now fixed?

Reply:
The slowdown is on average slightly less than 0.1 seconds per test (except for a few tests with many permutations).

`runnable/test17338.d` was the only exception with more than 80-fold slowdown, therefore `-L/OPT:NOICF` is now explicitly specified as a `REQUIRED_ARGS`.